### ISSUE TITLE
Multiline footnote doesn't work 

### ISSUE THREAD

**Issue by ApayRus** [hackmdio/hackmd-io-issues#214]
If a footnote has more than one line, only the first line becomes footnote, and other display like ordinary text in paragraph above footnote section. 

Is there a way to write multiline footnotes? 

Here is reproduction of the problem: https://hackmd.io/@aparus/ByFUVuFOv

**Comment by jackycute** [comment]
Do you find solution for this issue @Aparus?
The example you provided is no longer accessible.

**Comment by ApayRus** [comment]
> Do you find solution for this issue @Aparus?
> The example you provided is no longer accessible.

No, I didn't. Sorry, I deleted the example a couple of days ago :(

**Comment by ApayRus** [comment]
> Do you find solution for this issue @Aparus?
> The example you provided is no longer accessible.

Do you need it? I can reproduce it. It's not hard for me.

**Comment by jackycute** [comment]
> > Do you find solution for this issue @Aparus?
> > The example you provided is no longer accessible.
> 
> Do you need it? I can reproduce it. It's not hard for me.

Please do, it helps if you can give example code in the comment directly, thanks!
Also, if this issue is spec related, it's harder for us to propose changes.
